site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala

```diff
@@ -35,8 +36,8 @@ class ApplicationController(userService: UserServices)(implicit exerciseOps: Exe
     exerciseOps.getLibraries.runTask match {
       case \/-(libraries) ⇒
         request.session.get("oauth-token") map { token ⇒
-          val user = userService.getUserByLogin(GetUserByLoginRequest(login = request.session.get("user").getOrElse("")))
-          user.map(response ⇒ Ok(views.html.templates.home.index(user = response.user, libraries = libraries)))
+          val user = userService.getUserByLogin(request.session.get("user").getOrElse(""))
+          Future.successful(Ok(views.html.templates.home.index(user = user, libraries = libraries)))
```

Comment:
I think so, we can have anonymous users navigate the site with no problem. This won't allow exercise evaluation but the content is rendered so web crawlers can index it.